environment.yml

```diff
@@ -4,14 +4,14 @@ channels:
   - symbiflow
   - litex-hub
 dependencies:
-  - symbiflow::symbiflow-yosys=0.8_6021_gd8b2d1a2=20200708_083630
-  - symbiflow::symbiflow-yosys-plugins=1.0.0.7_0174_g5e6370a=20201012_171341
-  - symbiflow::symbiflow-vtr=8.0.0.rc2_5097_gf1a3bcc2a=20200916_072439
-  - symbiflow::zachjs-sv2v=0.0.5_0018_ga170536
-  - symbiflow::openocd
-  - symbiflow::iverilog
-  - symbiflow::icestorm
-  - symbiflow::capnproto-java
+  - litex-hub::symbiflow-yosys
```

Comment:
Regardless of whether the packages come from symbiflow or litex-hub channel, removing version pinning right now will result in unstable builds.  What is the plan to deal with incompatible packages?  I'm specifically concerned with VTR and yosys packages, as those are the ones mostly likely to break.